### DISCUSSION TITLE
Is using this API legal?

### DISCUSSION THREAD

**Discussion by codyandersan** [cyberboysumanjay/JioSaavnAPI#58] [Q&A] [answered]
I am thinking to make a Music Player and Downloader website using this API. I don't want to put any ads in my website and I do not intend to monetise it in any way.

I don't think this is the official API by JioSaavn.. (correct me if I am wrong)

I just want to know whether using this API is **legal** and will I run into any copyright infringement cases by using it?

I will be running my website over Github Pages.

**Comment by Suraj-Gov** [comment, marked as answer]
This isn’t an official api. It was developed by inspecting the api calls made by the jiosaavn site when a user wasn’t logged in. 

Using this api to allow downloading music is definitely not legal. It is against the T&Cs of jiosaavn.

**Reply by codyandersan** [reply]
Ok thanks, but if I do not monetize the website and clearly state that the data is taken from JioSaavn, then too would it be a problem for JioSaavn? 

And also, this API has been there for a long time, so JioSaavn should be knowing about those flaws in their website..

(And Musify too is based on this API, AFAIK 😅)

**Reply by Suraj-Gov** [reply]
https://www.jiosaavn.com/corporate/terms/

You can give it a shot, but you’re liable to be sued at any time. 
Not to rain on your parade, but just build what you want; after you consider if the benefits do outweigh the consequences.

**Reply by codyandersan** [reply]
> https://www.jiosaavn.com/corporate/terms/
> 
> You can give it a shot, but you’re liable to be sued at any time. 
> Not to rain on your parade, but just build what you want; after you consider if the benefits do outweigh the consequences. 

Ok thanks, I I'll look into this.
